packages/py-api/py_api/controllers/logs_controller.py

```diff
@@ -0,0 +1,30 @@
+from os.path import exists
+from pathlib import Path
+from typing import Final
+
+from fastapi.responses import FileResponse, JSONResponse
+from py_api.environment import IS_OFFLINE
+
+LOGFILE_PATH: Final = f"{Path(__file__).parent.resolve().parent}/shared/logfile.log"
+
+
+class LogsController:
+    @classmethod
+    def get_log_file(cls) -> JSONResponse | FileResponse:

```

Comment:
Since you don't call any class functions, drop the `@classmethod` decorator here.

Reply:
ooor put encapsulate the LOGFILE_PATH variable in the class itself and accessing by using `cos.LOGFILE_PATH`

Reply:
I would prefer the second but it's up to you.

Reply:
@asynchroza Yes but in a separate issue.

Reply:
@IvanObreshkov then create one and I will approve this PR.